packages/dds/tree/README.md

```diff
@@ -320,3 +320,24 @@ flowchart
     end
     package-->runtime["Fluid runtime"]
 ```
+
+# Open Design Questions
+
+## How should indexes relate to branches?
+Some possible options:
+
+ - Use copy on write in indexes, and keep all needed indexes for all needed revision within edit-manager. Provide all relevant indexes to change rebaser. Maybe allow change rebaser to compute intermediate indexes as needed.
```

Comment:
```suggestion
 - Use copy on write in indexes, and keep all needed indexes for all needed revisions within edit-manager. Provide all relevant indexes to change rebaser. Maybe allow change rebaser to compute intermediate indexes as needed.
```

Also nit: edit-manager having a hyphen but not change rebaser is a bit weird to me. I'm also wondering if they should both be referred to by class name e.g. `ChangeRebaser`.

Reply:
This is referring to edit-manager, the folder/module. It could instead refer to EditManager, the class, but this file (the top level readme) talks about these modules by their module name in the diagrams above, so I figured it would be more clear this way. ChangeRebaser should be fixed though.
```suggestion
 - Use copy on write in indexes, and keep all needed indexes for all needed revisions within edit-manager. Provide all relevant indexes to `ChangeRebaser`. Maybe allow `ChangeRebaser` to compute intermediate indexes as needed.
```